src/Contact/LocalRelationship/Repository/LocalRelationship.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2021, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Contact\LocalRelationship\Repository;
+
+use Friendica\Contact\LocalRelationship\Entity;
+use Friendica\Contact\LocalRelationship\Exception;
+use Friendica\Contact\LocalRelationship\Factory;
+use Friendica\Core\L10n;
+use Friendica\Database\Database;
+use Friendica\Network\HTTPException;
+use Psr\Log\LoggerInterface;
+
+class LocalRelationship extends \Friendica\BaseRepository
+{
+	protected static $table_name = 'user-contact';
+
+	/** @var Factory\LocalRelationship */
+	protected $factory;
+	/** @var L10n */
+	protected $l10n;
```

Comment:
I don't find any usage of this property

Reply:
It's in an upcoming part of this refactoring, I can remove it for now.